jenkins-schema.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash -xe
+export DISPLAY=:99
+export GOVUK_APP_DOMAIN=test.gov.uk
+export GOVUK_ASSET_ROOT=http://static.test.gov.uk
+export GITHUB_STATUS_CONTEXT="verify government-frontend against content format examples"
```

Comment:
I would capitalise the start of the sentence to match other build statuses we have.